src/dvr/dvr_db.c

```diff
@@ -4185,6 +4230,35 @@ dvr_entry_cancel(dvr_entry_t *de, int rerecord)
   return de;
 }
 
+static void
+dvr_entry_add_previously_recorded(dvr_entry_t *de)
+{
+  de->de_dont_reschedule = 1; // Set as not reschedule
+  de->de_file_removed = 1; // Set as file removed
+  de->de_start = gclk(); // Need in case you want to record it again after.
+  de->de_stop = gclk() + 1000; 
```

Comment:
I don't quite understand the changing of de_start/de_stop.

Reply:
I do it because I don't like to have removed recordings with a date later than the day I have omitted them, is only a cosmetic fix. :-)